build/KoreBuild.sh

```diff
@@ -69,6 +69,12 @@ if [ ! -d $sakeFolder ]; then
     mv "$toolsProject" "$toolsProject.norestore"
 fi
 
+nugetPath="$koreBuildFolder/nuget.exe"
+if [ ! -f $nugetPath ]; then
+    nugetUrl="https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"
```

Comment:
libuv-build repo needs it to build the nuget packages. If we really wanted we might be able to use dotnet for this but it would be very artificial - the package being built contains only native libs and at the moment there is not a single project.json there or even no code since we just pull libuv code as a submodule.